Configurations/README.design

```diff
@@ -498,7 +498,7 @@ etc.
 
 Along with the build-file templates is the driving engine
 Configurations/common.tmpl, which looks through all the information in
-%unified_info and generates all the rulesets to build libraries,
+%unified_info and generates all the rules-set to build libraries,
```

Comment:
nope, the original is correct, unless you want to replace with "rules" or "set of rules"